packages/components/src/range-control/index.tsx

```diff
@@ -220,11 +230,11 @@ function RangeControl(
 					marks={ !! marks }
 				>
 					<InputRange
-						{ ...props }
+						{ ...otherProps }
 						className="components-range-control__slider"
 						describedBy={ describedBy }
 						disabled={ disabled }
-						id={ id }
+						id={ id as string }
```

Comment:
Probably safer to make sure that `id` is an actual string?

```suggestion
						id={ `${id}` }
```